charts/prometheus-alerts/templates/containers-prometheusrule.yaml

```diff
@@ -120,9 +144,19 @@ spec:
       expr: |-
         sum by (namespace, pod) (
           max by(namespace, pod) (
-            kube_pod_status_phase{job="kube-state-metrics", namespace=~"{{ $targetNamespace }}", phase=~"Pending|Unknown"}
-          ) * on(namespace, pod) group_left(owner_kind) topk by(namespace, pod) (
-            1, max by(namespace, pod, owner_kind) (kube_pod_owner{owner_kind!="Job"})
+            kube_pod_status_phase{
+              job="kube-state-metrics",
+              phase=~"Pending|Unknown",
```

Comment:
Do we want `Failed` as well ?

Reply:
Maybe?... right now I don't want to change it too much..  We could make it customizable.

Reply:
Actually no.,. I think the purpose of this alarm is to tell you that the job isn't running. Not to tell you whether or not your own code failed in the job.